lib/middleware/ping/ping.js

```diff
@@ -1,33 +1,19 @@
 require('@ministryofjustice/module-alias/register-module')(module)
 
-const { APP_SHA, SERVICE_SHA, ROUTES } = require('~/fb-runner-node/constants/constants')
-const pingPayload = {
+const {
   APP_SHA,
   SERVICE_SHA
-}
+} = require('~/fb-runner-node/constants/constants')
 
-const init = (options = {}) => {
-  const PING_URL = options.url || ROUTES.ping
-  return (req, res, next) => {
-    if (req.originalUrl === PING_URL) {
-      ping(req, res)
-    } else {
-      next()
-    }
-  }
+const payload = {
+  APP_SHA,
+  SERVICE_SHA
 }
 
-const ping = (req, res) => {
-  res.json(pingPayload)
+function init () {
```

Comment:
I've simplified this down

There is no need to export the function `ping` since it isn't being used elsewhere. Nor does it need to exist

The `options` argument is never used. Only the unit tests were passing in an `options` argument with an `url` field. I've stripped all of this out

The `init` function in this version simply wraps `res.json()` and supplies a scoped variable